docs/doc_sources/beginners_guides/installation.rst

```diff
@@ -159,13 +159,32 @@ The following plugins from CodePlay are supported:
 .. _codeplay_nv_plugin: https://developer.codeplay.com/products/oneapi/nvidia/
 .. _codeplay_amd_plugin: https://developer.codeplay.com/products/oneapi/amd/
 
-``dpctl`` can be built for CUDA devices as follows:
+``dpctl`` can be built for CUDA devices using the ``DPCTL_TARGET_CUDA`` CMake option,
+which accepts a specific compute architecture string:
+
+.. code-block:: bash
+
+    python scripts/build_locally.py --verbose --cmake-opts="-DDPCTL_TARGET_CUDA=sm_80"
+
+To use the default architecture (``sm_50``),
+set ``DPCTL_TARGET_CUDA`` to a value such as ``ON``, ``TRUE``, ``YES``, ``Y``, or ``1``:
 
 .. code-block:: bash
 
     python scripts/build_locally.py --verbose --cmake-opts="-DDPCTL_TARGET_CUDA=ON"
 
-And for AMD devices
+Note that kernels are built for ``sm_50`` by default, allowing them to work on a wider
+range of architectures, but limiting the usage of more recent CUDA features.
+
+For reference, compute architecture strings like ``sm_80`` are based on
+CUDA Compute Capability. A complete mapping between NVIDIA GPU models and their
+respective ``sm_XX`` values can be found in the official
+`CUDA GPU Compute Capability <https://developer.nvidia.com/cuda-gpus>`_.
```

Comment:
The mapping is not clear from the reference doc.

Reply:
Yes, seems they aren't necessarily related either ([see here](https://docs.nvidia.com/cuda/volta-compatibility-guide/index.html#building-volta-compatible-apps-using-cuda-8-0) and below it)

Reply:
A CUDA developer [notes](https://forums.developer.nvidia.com/t/how-should-i-use-correctly-the-sm-xx-and-compute-xx/219160/2) that `sm_XX` refers to machine code for a specific GPU hardware architecture. Since each Compute Capability version corresponds to a particular architecture (CC 8.0 -> Ampere A100) it is reasonable to say that `sm_80` corresponds to `CC 8.0`

I changed the text a bit